simplq/src/components/common/utilFns.js

```diff
@@ -29,3 +29,5 @@ export const smoothScrollToHomePageTop = () => {
     window.location.href = '/';
   }
 };
+
+export const title = (txt) => txt?.charAt(0)?.toUpperCase() + txt?.substr(1)?.toLowerCase();
```

Comment:
https://stackoverflow.com/questions/7225407/convert-camelcasetext-to-sentence-case-text